src/BaGet/Startup.cs

```diff
@@ -90,20 +94,24 @@ private void ConfigureStorageProviders(IServiceCollection services)
                         return provider.GetRequiredService<BlobPackageStorageService>();
 
                     default:
-                        throw new InvalidOperationException(
-                            $"Unsupported storage service: {options.Storage.Type}");
+                        //this exception is only thrown if there is a not implemented value on the enum "StorageType"

```

Comment:
>this [](start = 26, length = 4)

Please capitalize comments and ensure there's a space between the `//` and the comment